dashboard/app/models/frequency_analysis.rb

```diff
@@ -0,0 +1,13 @@
+class FrequencyAnalysis < Widget
```

Comment:
I think this will happen whenever any users have progress in a script containing that level (not just the level itself).

If this level is not in any scripts that the public has access to it's probably not worrying about (it's only for the few minutes between rake seed and frontend upgrades anyway).